Assets/GothicVR/Scripts/Manager/GvrSceneManager.cs

```diff
@@ -159,6 +160,7 @@ private void SetLoadingTextureForWorld(string worldName, bool newGame = false)
 
         private void HideLoadingScene()
         {
+            SkyManager.I.InitSky();
```

Comment:
Worth a featureflag?

Reply:
Nope. But a performance flag. Which would need to work together with VobCollider. --> Something for another branch if needed for performance.